sample1.py

```diff
@@ -35,6 +38,7 @@ def computeIncreasingPairRatio(numberList):
     increasingPairRatio = float(numberOfIncreasingPairs) / float(numberOfAllPairs)
     return increasingPairRatio
 
-numberList = inputNumbers()
-increasingPairRatio = computeIncreasingPairRatio(numberList)
-print(str(increasingPairRatio))
+if __name__=="__main__":
```

Comment:
Python-ban sztenderd idiom a fő futtatandő részt egy ilyen blokkba tenni, hogy használható legyen a fájl könnyen máshonnan is. https://docs.python.org/3/library/__main__.html